runtime/altair/src/lib.rs

```diff
@@ -1532,7 +1532,7 @@ impl<
 
 parameter_types! {
 	pub const MaxKeys: u32 = 10;
-	pub const DefaultKeyDeposit: Balance = 100 * CFG;
+	pub const DefaultKeyDeposit: Balance = 100 * AIR;
```

Comment:
I don't remember if I've already talked with someone about it, but maybe it can be worth it to generalize our constants and use the same CFG for all runtimes. Then, each chain will call it as they want, but internally they can be the same constants.

Reply:
Actually, they are already [the same constant internally](https://github.com/centrifuge/centrifuge-chain/blob/08fa74fc31e7595bb25453dae6aecff1208ad662/runtime/altair/src/constants.rs#L25-L26) 😅 So this was just a style/consistency change.

Reply:
Purely copy and paste on my part :).

It's nice to keep them separate in case we do adjust our decimals, ex when AIR goes to the moon ;)

Reply:
Purely copy and paste on my part :).

It's nice to keep them separate in case we do adjust our decimals, ex when AIR goes to the moon ;)

Reply:
But being Altair a "test" of Centrifuge net, does it make sense to have a different decimal approach? I mean, the idea of this is to ensure later things will work well in the real network. If we differ on it, maybe could appear some minor precision bug no tested later in centrifuge.

Reply:
@lemunozm you are right in theory, but in practise often that happens. For example, `DOT` is a 10-decimal token while `KSM` is 12. The point here is that it can be happen that need arises and given that internally they match to the same constant, I'd leave this as is to keep one in mind that they are different assets in reality. Also, keep in mind that soon ™️  cross-network transfers will be possible, so we could have AIR being transferred to Centrifuge and CFG to Altair.